app/graphql/types/opening_hours_type.rb

```diff
@@ -1,8 +1,15 @@
 module Types
   class OpeningHoursType < Types::BaseObject
-    field :day_of_week, String
-    field :opens, String
-    field :closes, String
+    description 'The open hours within a specific weekday'
```

Comment:
Times this Partner is open to the public

Reply:
I think you just described the field as accessed from partner. This data structure contains information about a specific week day